python/ui/test/test_dropdown_option_select.py

```diff
@@ -0,0 +1,23 @@
+from python.common.basebrowser import BrowserExample
+from python.ui.method.dropdown_page import Dropdown
+from python.ui.method.internet_home_page import Homepage
+
+
+class TestDropdown:
+
+    def test_select_dropdown_object(self):
+        browser = BrowserExample().browser
+
+        the_homepage = Homepage(browser)
+        the_dropdown = Dropdown(browser)
```

Comment:
Small opinon/recommendation: I would name you page instantiated variable with "page" in it. like the_dropdown_page as it make is easier to understand that is what you are accessing later in the code.